CHANGELOG.md

```diff
@@ -19,4 +19,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Now use the new `Base64Coder.encode` in order to encode to base64
 ### Removed
 - `cl.transbank.onepay.util.BASE64Encoder`
-- `cl.transbank.onepay.util.CharacterEncoder`
\ No newline at end of file
+- `cl.transbank.onepay.util.CharacterEncoder`
+
+## [1.2.0] - 2018-08-14
+### Added
+- Add new callbackUrl config param to Onepay. You could set it with the following setter:
+    `cl.transbank.onepay.Onepay.setCallbackUrl("a-callback-endpoint")`
+- Add new optional param `channel` to `Transaction.create`
```

Comment:
I think you forgot to add the appScheme change on the Changelog (do you really want to mantain the changelog manually?)

Reply:
Added. I integrate it becouse Stripe and Paypal use an CHANGELOG. I dont know if they mantain them manually but I think it is good idea to have it.